youtube_dl/extractor/common.py

```diff
@@ -211,6 +211,10 @@ class InfoExtractor(object):
                     specified in the URL.
     end_time:       Time in seconds where the reproduction should end, as
                     specified in the URL.
+    chapters:       A list of dictionaries, with the following entries:
+                        * "start_time" - The start time of the chapter in miliseconds
```

Comment:
+1 for using a consistent unit for various time-related fields. Doing so brings less possible confusion in code studying.